docs/release/trg-4/trg-4-08.md

```diff
@@ -0,0 +1,24 @@
+---
+title: TRG 4.08 - Multi-platform images
+---
+
+| Status | Created     | Post-History                              |
+|--------|-------------|-------------------------------------------|
+| Active | 01-Sep-2024 | Initial contribution                      |
+| Draft  | 20-Mar-2024 | Initial contribution                      |
+
+## Why
+
+Providing container images that support multiple CPU architectures enhances flexibility, compatibility, and efficiency across hardware devices.
+
+This is especially beneficial for supporting use cases like tutorials and local playgrounds.
+
+## Description
+
+Container images **must** be built for **amd64 and arm64** CPU architectures.
```

Comment:
```suggestion
Container images **should** be built for **amd64 and arm64** CPU architectures.
```

I generally appreciate the idea of having multiplatform builds, but I don't see a point in making this mandatory. If it's easily supported, sure, why not. But converting a hypothetical image build that is non-multiplatform-capable into a multiplatform one could be a lengthy (and probably unnecessary) process, since with Rosetta emulation M[1|2|3] Macs can run Amd64 images.

TL;DR: swap **must** with **should** and i'll approve it.

Reply:
@paullatzelsperger sure, I think that's fair for this topic, changed. thanks for reviewing!